relayer/chains/penumbra/provider.go

```diff
@@ -60,6 +60,7 @@ type PenumbraProviderConfig struct {
 	Modules          []module.AppModuleBasic    `json:"-" yaml:"-"`
 	Slip44           int                        `json:"coin-type" yaml:"coin-type"`
 	Broadcast        provider.BroadcastMode     `json:"broadcast-mode" yaml:"broadcast-mode"`
+	MaxMsgNum        uint64                     `json:"max-msg-num" yaml:"max-msg-num"`
```

Comment:
can we rename this to `BroadcastMaxMsg`, i think the name is more clear with regards to what the value is used for